README.md

```diff
@@ -1,70 +1,303 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
```

Comment:
[OPTIONAL] Kindly remove all commented codes from your project since it makes it bulky and can slow down the speed of your project as it increases.  :+1: